Packages/com.unity.inputsystem/InputSystem/Devices/Keyboard.cs

```diff
@@ -1006,6 +1034,25 @@ public event Action<IMECompositionString> onIMECompositionChange
         /// See <see cref="Keyboard.SetIMECursorPosition"/>, <see cref="Keyboard.onIMECompositionChange"/>,
         /// <see cref="Keyboard.imeSelected"/> for more IME settings and data.
         /// </remarks>
+        /// <param name="enabled">
+        ///     The new IME composition enabled state. True to enable the IME, false to disable it.
```

Comment:
Should "the new" be removed and instead just use "The IME composition..."

Reply:
agree

Reply:
Also I believe <c>true</c> and <c>false</c> when referencing keywords or identifiers I believe

Reply:
fixed in [8ad97b6](https://github.com/Unity-Technologies/InputSystem/pull/2069/commits/8ad97b6e1a384f347459dbfa06ba72c998464ce4)